test/ContractionOff.ll

```diff
@@ -18,17 +18,44 @@
 ; opt -mem2reg 1.ll -S -o 1.o.ll
 
 ; RUN: llvm-as %s -o %t.bc
-; RUN: llvm-spirv %t.bc -spirv-text -o %t
-; RUN: FileCheck < %t %s
-; RUN: llvm-spirv %t.bc -o %t.spv
-; RUN: spirv-val %t.spv
+; RUN: llvm-spirv %t.bc -spirv-text -o %t.default
+; RUN: FileCheck < %t.default %s --check-prefixes CHECK,CHECK-ON
+; RUN: llvm-spirv %t.bc -o %t.spv.default
+; RUN: spirv-val %t.spv.default
+
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv %t.bc -spirv-text -o %t.off -ffp-contract=off
+; RUN: FileCheck < %t.off %s --check-prefixes CHECK,CHECK-OFF
+; RUN: llvm-spirv %t.bc -o %t.spv.off -ffp-contract=off
```

Comment:
Personally, I prefer the following sequence of commands:

```
; RUN: llvm-spirv %t.bc -o %t.spv.off -ffp-contract=off
; RUN: llvm-spirv %t.spv.off -to-text -o - | FileCheck %s --check-prefixes CHECK,CHECK-OFF
```

It is not necessary to merge to-text and FileCheck, but the main idea is that you create binary SPIR-V representation first, and then convert it to a text one: the latter step shouldn't require "-ffp-contract" option, as it just changes the representation and doesn't perform actual conversion

Reply:
Done.